pkg/generator/appconfiguration/generators/components_generator.go

```diff
@@ -9,23 +9,25 @@ import (
 
 type componentsGenerator struct {
 	projectName string
+	stackName   string
 	components  map[string]component.Component
 }
 
-func NewComponentsGenerator(projectName string, components map[string]component.Component) (Generator, error) {
```

Comment:
Why is component still needed?

Reply:
The architecture of the generator has been updated.